### PR TITLE
docs: :memo: fiori elements odata v4 test library integration

### PR DESCRIPTION
**Pull request by georgbischoff**
documentation how to integrate the Fiori Elements OData v4 test library in wdi5

### CODE REVIEW

**Comment by vobu** [comment]
so cool 🚀 
thanks for getting onto this - can't wait until it is `ready for review` and we can get this goodie in

File: docs/fe-testlib.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+Nevertheless, you need to implement certain steps to test the specific functionalities of the app, e.g. loading data into the list-report, navigating between a list-report and an object-page, saving business objects, etc. To implement these steps within a WDI5 test, you can reuse the test functions of the OData V4 OPA test library and you don´t need to re-implement these functions. This saves time and avoids redundancy.
+
+Another advantage of using the OData V4 test library is that updates for the test functions due to framework changes are automatically provided by the framework. This ensures that your tests will not break.
+
+### Main benefits of using the test library
+
+- Reduces test maintenance efforts and avoid code repetition.
+- Isolate generic actions and validations in the test library and reuse them across apps.
+- Tests are simplified and have compact page objects and short journeys.
```
Comment (philippschardt) [inline docs/fe-testlib.md:17]: Journeys are not necessarily short when using the test lib

**Review by philippschardt — CHANGES_REQUESTED** [review]
You should be careful that you have a dedicated name for the test lib all over the document. name changes slightly from time to time
Otherwise looks good so far

File: docs/fe-testlib.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+- Further common test functions.
+- (JourneyRunner: Execute integration tests with given settings. - not used in wdi5)
+
+## Consistent naming of test functions
+
+Unified expressions improve the readability.
+
+- Actions: `i<DoSth><WithSth>`
+- Assertions: `i<ExpectSth>`
+- Consistent parameter signature, e.g. TODO
```
Comment (Siolto) [inline docs/fe-testlib.md:40]: Here is still a ToDo open

File: docs/fe-testlib.md (hunk 1 above)
Comment (georgbischoff) [reply docs/fe-testlib.md]: Removed as we also have no journeys in wdi5

File: docs/fe-testlib.md (hunk 2 above)
Comment (georgbischoff) [reply docs/fe-testlib.md]: Added two examples

**Review by Siolto — APPROVED** [review]
Looks good to me 👍 Thanks for the valuable input!!

**Review by philippschardt — APPROVED** [review]
Looks fine now

**Review by ARiesterer — APPROVED** [review]
thumbs up!

**Comment by vobu** [comment]
thanks so much for the collab on this one! you guys 🎸

**Pull request merged**